src/parser/jobs/war/modules/StormsEye.tsx

```diff
@@ -51,20 +54,20 @@ export default class StormsEye extends Module {
 	onComplete(): void {
 		this.checklist.add(new Rule({
 			name: <Trans id="war.stormseye.checklist.name">Keep Storm's Eye Up</Trans>,
-			description: <Trans id="war.stormseye.checklist.description">Storm's Eye increases your damage by 10%, it is a substantial part of a Warrior's damage.</Trans>,
+			description: <Trans id="war.stormseye.checklist.description">Storm's Eye increases your damage by 10%, a substantial part of a Warrior's damage.</Trans>,
```

Comment:
The phrasing on this still feels a tiny bit weird to me; if it's getting changed anyway, maybe "...by 10%, which makes up a substantial..."?

Reply:
I think I'll leave this one to 6.0 tbh, since I'm not sure that's better and I can't think of something clearer either.